### DISCUSSION TITLE
include track numbers in track names

### DISCUSSION THREAD

**Discussion by 594jjc** [Shabinder/SpotiFlyer#1240] [Q&A]
Hello. I play my MP3s in alphabetical order, and so I always add the track number to the file name as the first two characters of the file name. This way when I play in alphabetical order they play in the order on the album. When I download with Spotiflyer it just names the MP3s with the track titles, so I can't sort and play alphabetically... is there a way to have Spotiflyer append the track number onto the MP3 name when I download them? My Android MP3 player can sort by 'track number' which must be ID3 information. That works for now.

Other than that this software is SPOT ON. I just sent a donation via paypal bc you all deserve support. 

Thank you!!!

**Comment by MembershipExcel** [comment]
+1 for this
I'm surprise the filename is only the track title not something like : 
"# - Song Title - Artist Name - Album name"


In the meantime, i'm using a software called [Mp3Tag](https://g.co/kgs/s5EgLR). And on Android "AutomaTag".

**Comment by TomaszFH** [comment]
If you have a few versions of ex. "Fever" from different artists in your playlist then just the track name as a file name is not enough.
Please add the playlist track position to the file name. It would be great improvement for a start.
